### PR TITLE
Fix issue #62, sidebar for user reward

### PR DESCRIPTION
**Pull request by schalkneethling**
@chrismore I would need to know where to post the form or, whether there is some third party snippet that needs to be embedded. Other than that, see the screenshot below for the current look and feel:

screen shot 2016-06-27 at 16 19 38

### CODE REVIEW

**Comment by schalkneethling** [comment]
If the general layout and such works, as mentioned in the stand-up, I would like to get this merged so it can become part of the flow and I/we can also continue work on issues such as:

* https://github.com/mozilla/all-aboard/issues/159
* https://github.com/mozilla/all-aboard/issues/158
* https://github.com/mozilla/all-aboard/issues/157

**Comment by chrismore** [comment]
@schalkneethling check out this issue I just filed. https://github.com/mozilla/all-aboard/issues/178 We may want to do this now as the more that I think about it, it is a bit more safe.

**Comment by FrancescoSTL** [comment]
@schalkneethling I'll review for now and if it works, merge. @chrismore mentioned adding our own temporary iframe on mozilla.org to test, and then changing the url to redirect to the vendor's iframe when needed. See #178

File: data/css/all-aboard.css [hunk 1]
```diff
@@ -284,3 +321,15 @@ a:focus {
     height: 0;
     opacity: 0;
 }
+
+/* Hide visually, but not to screen readers */
+.visually-hidden {
```
Comment (FrancescoSTL) [inline data/css/all-aboard.css]: @schalkneethling should this be denoted as visually-hidden? It seems to be showing up fine

**Comment by FrancescoSTL** [comment]
Other than one comment, r+. Feel free to merge if `.visually-hidden` has some meaning

**Comment by chrismore** [comment]
As mentioned on slack, the form in this sidebar, should be replaced with an iframe to an interstitial page on www.mozilla.org as seen with in issue #178. All of the contents of the sidebar should be local to the sidebar and the iframe will simply replace the two form fields and submit button.

File: data/css/all-aboard.css (hunk 1 above)
Comment (schalkneethling) [inline data/css/all-aboard.css]: This is used to hide the form label elements from visual users but, it will still be accessible to screen reader users.

**Comment by FrancescoSTL** [comment]
@schalkneethling we will probably want to create an issue to remove the unnecessary code once the iframe is built on moz.org

**Comment by chrismore** [comment]
@schalkneethling r+ merge for now to complete the overall experience.

**Pull request merged**